### PR TITLE
TASK: Use new generation version of editorconfig-checker

### PR DESCRIPTION
**Pull request by mstruebing**
Hey, I've completely rewritten the editorconfig-checker in go and only use a wrapper for other languages now.
However, as this was run on my machine only until now I've not published the JS version as stable. I wanted to try it out on a real and bigger project so I've used this one  :)

It would be **super** cool if some of you contributors could check out the PR and run a `make install && make lint-editorconfing` to verify it is not only working on my machine.

**TLDR;**

1. Check out PR
2. Run `make install`
3. Run `make lint-editorconfig`

Expected behaviour:

1. Nothing.

This should not output anything and not throw any error message on you. :)

### CODE REVIEW

**Comment by markusguenther** [comment]
Looks  good to me :) 👍

**Comment by dimaip** [comment]
Hi! What does it mean that you've re-written it in Go? Does it require go runtime to run it then?
Sorry, don't have the energy to try it locally now...

**Comment by mstruebing** [comment]
Nope! That's the power of go :) - NO RUNTIME DEPENDENCIES NEEDED.

That's the core: https://github.com/editorconfig-checker/editorconfig-checker
And the Javascript-Wrapper(currently bundled into this PR: https://github.com/editorconfig-checker/editorconfig-checker.javascript/pull/47) downloads the correct binary of the go-version for your system and runs this instead.

I'm a bit afraid to release this finally so I wanted to let some others test it before. But it seems everything works fine \o/

**Comment by mstruebing** [comment]
I've released it right now.
Would be cool if you would test/use it and see if it works :)

**Comment by markusguenther** [comment]
bildschirmfoto 2019-02-27 um 20 55 16

And again ... everything is fine on Mac 👍

**Comment by mstruebing** [comment]
:tada:
